api/Dockerfile

```diff
@@ -8,7 +8,8 @@ RUN apk --no-cache add build-base \
     && apk --no-cache add postgresql-dev \
     && pip install --upgrade pip \
     && pip install --no-cache-dir -r requirements.txt \
-    && pip install gunicorn==19.9.0
+    && pip install gunicorn==19.9.0 \
+    && echo 'alias ll="'ls -lah'"' >> /etc/profile
```

Comment:
What does this line do? Is it just setting up a shorthand command in the container?

Reply:
Right. Just an optional convenience. I got used to that alias so `ll` is muscle memory for me instead of `ls`. E.g. Ubuntu has that alias by default. The normal way of shelling in doesn't load the profile though so you have to use the -l option `docker container exec -it meep-backend_api_1 /bin/ash -l`. I can survive without it of course, but figured it was fine at the dev stage.

Reply:
Added clarifying comments for this in my last commit.